lms/djangoapps/django_comment_client/management/commands/export_discussion_participation.py

```diff
@@ -153,7 +153,10 @@ def dump_all(self, *args, **options):
                 raw_course_key,
                 dir_name / self.get_default_file_location(raw_course_key)
             ]
-            self.dump_one(*args, **options)
+            try:
+                self.dump_one(*args, **options)
+            except CommandError as e:
```

Comment:
@antoviaque nit: it would be nice if it could only catch exceptions caused by missing cohorted configs. The simpliest way would be to introduce `MissingCohortedConfigException` subclassed from CommandException and throw if not cohorted config is available.